### PR TITLE
Added ntosext extensions in the GettingStarted.md document.

### PR DESCRIPTION
**Pull request by shpalani**
## Description
 - ntosext extensions added in the GettingStarted.md

## Testing

_Do any existing tests cover this change? Are new tests needed? N/A

## Documentation

_Is there any documentation impact for this change? Yes

## Installation

_Is there any installer impact for this change? N/A

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -595,3 +595,4 @@ development cycle.  See the [Windows Hardware Developer documentation](https://l
 ## eBPF external extensions
 Extensions that integrate with eBPF for Windows:
 - XDP extension: [xdp-for-windows](https://github.com/microsoft/xdp-for-windows)
+- NTOS eBPF extensions: [ntosebpfext](https://github.com/microsoft/ntosebpfext)
```
Comment (dthaler) [inline docs/GettingStarted.md:598]: There are two separate extensions there that I'd prefer be listed separately here with a link to their own readme file.
ntosebpfext however has no README file and so isn't really documented (this is https://github.com/microsoft/ntosebpfext/issues/95).
And neteventebpfext is more about network events than "NTOS" per se, so calling it an NTOS extension is a misnomer in my view.  If I were looking for network events, I wouldn't look under "NTOS" extensions, and so wouldn't find it.

Comment (shpalani) [reply docs/GettingStarted.md]: Thanks for the suggestion. Until the readme is committed, I have pointed ntosebpfext to the main page.

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -595,3 +595,6 @@ development cycle.  See the [Windows Hardware Developer documentation](https://l
 ## eBPF external extensions
 Extensions that integrate with eBPF for Windows:
 - XDP extension: [xdp-for-windows](https://github.com/microsoft/xdp-for-windows)
+- NTOS eBPF extensions:
+   - ntosebpfext extension: [ntosebpfext](https://github.com/microsoft/ntosebpfext)
+   - Network event extension: [neteventebpfext](https://github.com/microsoft/ntosebpfext/blob/main/docs/neteventebpfext.md)
```
Comment (dthaler) [inline docs/GettingStarted.md:600]:
```suggestion
- ntosebpfext extension: [ntosebpfext](https://github.com/microsoft/ntosebpfext)
- Network event extension: [neteventebpfext](https://github.com/microsoft/ntosebpfext/blob/main/docs/neteventebpfext.md)
```
As I mentioned in my previous comment, I don't think "network event extension" is really an "NTOS" extension (or calling it such is confusing in any case)

File: docs/GettingStarted.md [hunk 3]
```diff
@@ -595,3 +595,5 @@ development cycle.  See the [Windows Hardware Developer documentation](https://l
 ## eBPF external extensions
 Extensions that integrate with eBPF for Windows:
 - XDP extension: [xdp-for-windows](https://github.com/microsoft/xdp-for-windows)
+- ntosebpfext extension: [ntosebpfext](https://github.com/microsoft/ntosebpfext)
```
Comment (matthewige) [inline docs/GettingStarted.md:598]: For the extensions in the ntos repo, should we be pointing to the code, or the readme for that extension, or what exactly? This first link is the entire github repo itself, but the extension is actually only a subfolder.

Maybe we should link to:
https://github.com/microsoft/ntosebpfext/tree/main/ebpf_extensions/ntosebpfext and then the netevent one should be https://github.com/microsoft/ntosebpfext/tree/main/ebpf_extensions/neteventebpfext?

Comment (shpalani) [reply docs/GettingStarted.md:598]: > For the extensions in the ntos repo, should we be pointing to the code, or the readme for that extension, or what exactly?

Currently, the readme for ntosebpfext process creation is tracked in a separate issue. Once that is done, we can always update here. Until then, we are pointing to the main page.

**Review by matthewige — APPROVED** [review]

**Review by dthaler — APPROVED** [review]

**Pull request merged**